bin/cml/runner.js

```diff
@@ -375,6 +376,24 @@ const run = async (opts) => {
     process.exit(0);
   }
 
+  if (reuseIdle) {
+    if (driver === 'bitbucket') {
+      winston.error('cml runner flag --reuse-idle is unsupported by bitbucket');
+      process.exit(1);
```

Comment:
It shouldn't have tried to create an instance yet, I felt that a message would be better since the problem is usage related. throwing an `Error` may distract a user with the stack trace thinking there is a bug?